feat: Update ticket links and enhance countdown timer

vtushar06 opened the pull request:
<!-- In order to keep off topic discussion to a minimum, it helps if the "work to be done" is already agreed on. -->
<!-- Ideally, a GitHub Issue with the label `Status: Consensus` will have been concluded, and linked to in this PR. -->
<!--
Thanks for submitting a pull request! Please provide enough information so that others can review your pull request.
-->

**What kind of change does this PR introduce?**
Feature enhancement and bug fix

<!-- E.g. a bugfix, feature, refactoring, etc… -->

**Issue Number:**
<!-- Pick one of the below options.  Please remove those which don't apply. -->
-  Closes #79   <!-- Replace ___ with the issue number this PR resolves -->

**Screenshots/videos:**
https://drive.google.com/file/d/1mRgAwfFFe4NuCd1gm8yajbY2M-1wetde/view?usp=drive_link
<!--Add screenshots or videos wherever possible.-->

**If relevant, did you update the documentation?**

<!--Add link to it-->

**Summary**

<!-- Explain the motivation for making this change. What existing problem does the pull request solve? -->
<!-- Try to link to an open issue for more information. -->

1. Enhanced Countdown Timer:

      - Modern gradient background with animations and responsive
      - Proper flexbox layout for centered numbers and labels.

2. Added 2025 Sponsors:

    - Sourcemeta and .txt as platinum sponsors
    - dev.events as media partner

3. Unified Sponsor Display:

    - Homepage now matches sponsors page (same HTML/CSS structure)
    - Consistent card design (200px width)
    - Updated ticket links to correct APIdays URL


**Does this PR introduce a breaking change?** No

<!-- If this PR introduces a breaking change, please describe the impact and a migration path for existing applications. -->

vtushar06 committed:
feat: Update ticket links and enhance countdown timer

- Updated ticket link for JSON Schema Conference 2025 in list.html and tickets.html.
- Enhanced countdown timer in countdown.html and countdown.js with animations for value changes.
- Improved sponsors section in sponsors-list.html with a carousel layout for platinum and media partners.
- Added new SVG logos for Sourcemeta and media partners.
- Styled sponsors sections for better visual appeal and responsiveness.

jagpreetrahi commented:
@vtushar06 you did a good job so far. But I would like to suggest some fixes into UI/UX as.
 Could you please minimize the padding for countdown value to outer box. 
And have you done with the changes which I mentioned earlier regarding a button that overlap to the navbar.??

vtushar06 commented:
> @vtushar06 you did a good job so far. But I would like to suggest some fixes into UI/UX as. Could you please minimize the padding for countdown value to outer box. And have you done with the changes which I mentioned earlier regarding a button that overlap to the navbar.??

Hii @jagpreetrahi, okay I will update as per your instructions, and for that Button overlapping I found that it has more things to fix up, so I thought I will fix those under another PR so you can manage them easily.

Utkarsh-123github reviewed (changes requested):
Hi @vtushar06 , I want to suggest some changes here:
- Please set the design such that numbers should be in center and not shifted downwards.
<img width="652" height="152" alt="image" src="https://github.com/user-attachments/assets/f649ad25-f549-4238-9ec7-87f1f748e09b" />

- I don't think there is any need of changing the sponsors UI.
<img width="531" height="198" alt="image" src="https://github.com/user-attachments/assets/9f10c03e-af7d-40fa-9d80-d383d5dc0193" />

vtushar06 committed:
updated sponsors to original look

claude committed:
Enhance countdown timer with gradient styling

- Applied blue gradient background (90deg, #1e5799 → #2989d8 → #207cca)
- Changed to pill-shaped design with 50px border-radius
- Added semi-transparent time boxes with backdrop blur effect
- Increased font size for numbers to 2.2rem for better visibility
- Added smooth hover effects with subtle lift animation
- Improved letter-spacing and font weights for modern look
- Maintained responsive design for all screen sizes

claude committed:
Clean up countdown timer structure and improve styling

- Remove unnecessary countdown-number-box divs for cleaner structure
- Numbers and labels now directly in countdown-item for better centering
- Increase font sizes for better visibility (2.4rem for numbers)
- Improve padding and spacing for balanced appearance
- Update labels to uppercase (DAYS, HOURS, MINUTES, SECONDS)
- Optimize responsive design for mobile and tablet
- Simplify CSS by removing unused styles

vtushar06 committed:
corrected alignment of countdown timer

vtushar06 commented:
Hii @Utkarsh-123github, can you please look into this. If any more changes required please let me know.

Honyii commented:
@Utkarsh-123github pls can you have a look at this today so we can close latest tomorrow.

Utkarsh-123github commented:
Hi @vtushar06 , why are these sponsors icons of Sourcemeta and txt so big ? 
Please adjust it to size of Octue and dev events icon to maintain consistency throughout.
Also, the countdown is not working or am I missing something?
<img width="1894" height="628" alt="image" src="https://github.com/user-attachments/assets/b0257591-add9-43cc-b2da-66bf5d8d080c" />
<img width="1147" height="464" alt="image" src="https://github.com/user-attachments/assets/fe020294-e31e-4b34-b5f2-2620e167305a" />

Utkarsh-123github reviewed (changes requested):
Please check my latest comment @vtushar06 .
Thank you!

vtushar06 commented:
yes, I will make changes asap, can you please have a look issue #86 , I have just shared video link after applying changes.

vtushar06 committed:
feat: Update sponsor card styles and improve countdown script path

vtushar06 commented:
Hii @Utkarsh-123github, I have made changes accordingly but there may be slight issue with dimension they can be visible slight small but I have tried making them as original but there are several issues with css so that why I am moving forward with that.
please let me know if you have more changes in mind to have I will love to implement that.

Utkarsh-123github reviewed (approved):
Good work @vtushar06 🚀🚀 
@Honyii , I have approved this, you can go with the merge whenever you are free.
Thank you!
